src/eth/accounts/setup.js

```diff
@@ -40,15 +78,22 @@ export async function getBrowserProvider() {
   }
 }
 
+function getInfuraUrl(protocol = 'https', network, infuraApiKey) {
+  let url = `${protocol}://${network}.infura.io`;
+  url += protocol === 'wss' ? '/ws' : '';
+  url += infuraApiKey ? `/${infuraApiKey}` : '';
```

Comment:
when `v3` was left in the url, i got "Internal error", probably because the API key I have is an old one... if we want to require v3 API keys we'll need to make sure that everyone on the team gets one (it's ok to just share one)